FSharp.Directory.Build.props

```diff
@@ -46,6 +46,7 @@
 
   <!-- other -->
   <PropertyGroup>
+    <DebugType>full</DebugType>
```

Comment:
Summoning @tmat - I don't think this is the right thing to do.  Instead, produce portable PDBs for packages and local development, and use `pdb2pdb` to post-process those to windows pdbs for archiving.

Reply:
@pilchie Just chatted with Brett, there may be a bug in our portable pdb writting, where we emit them in an unexpected order.  Which messes with the translator.

Reply:
I've been talking with him and we emit the portable PDBs incorrectly so the converter tool fails.  I'm adding this to unblock VS insertions (symbol check failures) until we fix our PDB writing, which @KevinRansom and I are looking at now.